paddle/fluid/inference/api/analysis_config.cc

```diff
@@ -164,11 +164,12 @@ void AnalysisConfig::EnableMkldnnQuantizer() {
   Update();
 }
 
-std::shared_ptr<MkldnnQuantizerConfig> AnalysisConfig::mkldnn_quantizer_config()
-    const {
+// std::shared_ptr<MkldnnQuantizerConfig>
+// AnalysisConfig::mkldnn_quantizer_config()
```

Comment:
You can remove the unused lines, such as 167-168.

Reply:
Ok, Thanks!